src/hooks/index.tsx

```diff
@@ -75,7 +75,9 @@ export async function getFileContent(
   const branch = fileRef || "HEAD";
 
   const apiUrl = `https://raw.githubusercontent.com/${owner}/${repo}/${branch}/${path}`;
-  const res = await fetch(apiUrl);
+  const res = await fetch(apiUrl, {
+    headers: PAT ? { Authorization: `token ${PAT}` } : {},
+  });
```

Comment:
I tried to make this change in https://github.com/githubnext/blocks-template/pull/24/ but got CORS errors from `Authorization`, so I just dropped the `Accept` header that was there previously.